app/src/main/java/com/example/whatTravel/helpers/ImageHelper.java

```diff
@@ -0,0 +1,43 @@
+package com.example.whatTravel.helpers;
+
+import android.graphics.Bitmap;
+import android.support.v4.util.LruCache;
+
+import com.android.volley.toolbox.ImageLoader;
+import com.example.whatTravel.API.WTApiLoadManager;
+
+/**
+ * Created by jiecongwang on 2/14/15.
+ */
+public enum  ImageHelper {
+    INSTANCE;
+
+    private final ImageLoader imageLoader;
+    private ImageHelper() {
+        imageLoader = new ImageLoader(WTApiLoadManager.getInstance().getRequestQueue(),
+                new ImageLoader.ImageCache() {
```

Comment:
This is not actually indentation, this sentence is so long that Android studio seperates it into two line